### DISCUSSION TITLE
Discord chat

### DISCUSSION THREAD

**Discussion by martinvonz** [jj-vcs/jj#257] [General]
There's now a [discord server](https://discord.gg/dkmfj3aGQN) for the project. Welcome!

**Comment by erlend-sh** [comment]
Could you add it as a readme button? Example:
`[![Discord](https://img.shields.io/discord/691052431525675048.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/bevy)`

**Reply by martinvonz** [reply]
I tried in #497, but it says "widget disabled". Do you know what's wrong?

**Reply by erlend-sh** [reply]
https://shields.io/category/chat
> To use the Discord badge a Discord server admin must enable the widget setting on the server.

Maybe this?

**Reply by martinvonz** [reply]
Ah, that's what it was. Thanks!
